cmd/arrai/main.go

```diff
@@ -79,10 +80,19 @@ VERSION:
 	err := app.Run(args)
 	if err != nil {
 		logrus.Info(err)
-		if _, isContextErr := err.(rel.ContextErr); isContextErr {
-			if err = createDebuggerShell(err); err != nil {
-				logrus.Info(err)
+		if isTerminal() {
+			if _, isContextErr := err.(rel.ContextErr); isContextErr {
+				if err = createDebuggerShell(err); err != nil {
+					logrus.Info(err)
+				}
 			}
+		} else {
+			logrus.Info("Shell can not be started as you are not using a terminal")
```

Comment:
Prefer a more fine-grained report. `"unable to start debug shell: standard input is not a terminal"`, etc.